docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -257,11 +258,59 @@ let configuration = ApolloCodegenConfiguration(
 
 </MultiCodeBlock>
 
+#### Swift package
+
+**[`ModuleType.swiftPackage(apolloSDKDependency: ApolloSDKDependency)`](https://www.apollographql.com/docs/ios/docc/documentation/apollocodegenlib/apollocodegenconfiguration/schematypesfileoutput/moduletype-swift.enum/swiftpackage)**
+
+This option automates the creation of an SPM package for your schema types. This generates a `Package.swift` file that is suitable for linking the generated schema types files to your project using Swift Package Manager.
+
+The `apolloSDKDependency` paramter allows you to customize the `apollo-ios` dependency of your SPM package by providing a url and SDK version. The `url` parameter allows you to customize the location of the `apollo-ios` dependency used by the generated SPM package, and has a default value that points to the main [`apollo-ios`](https://github.com/apollographql/apollo-ios) repository. The `sdkVersion` parameter allows you to set the version of the `apollo-ios` dependency to use. The default value of this is the `default` case, which uses an exact version of `apollo-ios` matching the code generation version in use which is the recommended way of handling this. The other provided options allow flexibility for doing things such as testing experimental branches, or using local versions for testing.
```

Comment:
_I don't know why I can't make an inline edit (suggestion) for this change but here's the change._

From:
```
The default value of this is the `default` case, which uses an exact version of `apollo-ios` matching the code generation version in use which is the recommended way of handling this.
```
To: 
```
The default value is the `default` case, which uses an exact version of `apollo-ios` matching the code generation version in use. This is the recommended configuration.
```